circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
```

Comment:
Circle has an amazing SSH feature that lets you connect to a box after a failed run. I added this so that I could live-debug what turned out to the the eventlet-pytest-py3 compatibility issue (on my local box it just hangs).

Circle doesn't parallelise builds using tox though, so the test run is slower. I'm happy to remove this again, but I will keep it in a gist for the next time!